tests/foreman/api/test_contentview.py

```diff
@@ -5,9 +5,9 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: API
+:CaseComponent: ContentView
```

Comment:
Bugzilla component is called `ContentViews`, in plural. See testimony.yaml for reference.